src/main/kotlin/com/wire/bots/polls/setup/HttpClient.kt

```diff
@@ -32,23 +25,22 @@ fun createHttpClient(meterRegistry: MeterRegistry) =
         }
     }
 
-/**
- * Debug logger for HTTP Requests.
- */
-private val Logger.Companion.DEBUG: Logger
```

Comment:
What is this, and why is it gone? Is this the handler for logging at the DEBUG level? Does it mean we don't log DEBUG level anymore?

Reply:
Nope it was unused code, probably there just to switch log level of http logging locally while developing. Does not make sense to have 2 functions and use 1 at the time, better to use 1 and maybe use a env variable if need to switch (this was not the case)

Reply:
Thanks for clarifying!